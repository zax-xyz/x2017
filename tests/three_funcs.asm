FUNC LABEL 0
    MOV REG 0 VAL 1
    MOV REG 1 VAL 2
    MOV REG 2 VAL 10
    CAL VAL 1
    RET
FUNC LABEL 1
    PRINT REG 0
    PRINT REG 1
    PRINT REG 2
    ADD REG 1 REG 2
    ADD REG 0 REG 2
    CAL VAL 2
    RET
FUNC LABEL 2
    PRINT REG 0
    PRINT REG 1
    PRINT REG 2
    RET
