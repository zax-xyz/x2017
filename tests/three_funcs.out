1
2
10
11
12
10
